docs/topics/js/js-overview.md

```diff
@@ -59,6 +59,61 @@ at the expense of type safety.
 Kotlin/JS is also compatible with the most common module systems: UMD, CommonJS, and AMD. Being able to [produce and consume modules](js-modules.md)
 means that you can interact with the JavaScript ecosystem in a structured manner.
 
+## Kotlin/JS frameworks
+
+Modern web development benefits significantly from frameworks that simplify building web applications.
+Here are examples of popular web frameworks for Kotlin/JS written by different authors:
+
+### KVision
+
+_KVision_ is an object-oriented web framework that makes it possible to write applications in Kotlin/JS without having to
```

Comment:
Seems like a bizarre order of sentences...
Is it necessary to say what this framework doesn't provide/have first?

I see this approach in other framework descriptions (like Doodle), and I am not sure that this is a good way.